code/game/objects/items/weapons/tools.dm

```diff
@@ -62,6 +62,16 @@
 	toolspeed = 0.25
 	var/obj/item/weapon/screwdriver/power/counterpart = null
 
+/obj/item/weapon/wrench/rusted
+	desc = "This wrench seems to have seen a few many days."

```

Comment:
`seen a few many`

Reply:
...huh. That's... not grammatically accurate. My life is a lie. I'd seen it used in person and in books, I feel so lost now. Will fix, thanks.